include/kvdk/types.h

```diff
@@ -66,6 +66,7 @@ __attribute__((unused)) static char const* KVDKValueTypeString[] = {
   GEN(PMemMapFileError)     \
   GEN(InvalidBatchSize)     \
   GEN(TooManyAccessThreads) \
+  GEN(InvalidAccessThread)  \
```

Comment:
Appending new status codes at the end of list may be better for compatibility.
And there are two copies of these states in Java:
https://github.com/pmem/kvdk/blob/main/java/kvdkjni/kvdkjni.h#L93
https://github.com/pmem/kvdk/blob/main/java/src/main/java/io/pmem/kvdk/Status.java#L51